daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -54,6 +57,27 @@ Dapr can also run [Slim self-hosted mode]({{< ref self-hosted-no-docker.md >}})
 dapr init -s
 ```
 
+You can also specify a private registry to pull container images from. These images needs to be published to private registries in below fashion to let dapr init pull them successfully - 
```

Comment:
This image registry setting is in the self-hosted section of the docs. Does this not work on Kubernetes? I would have expected that you can set this there. Otherwise we need to explicitly say this, but IMO that is a bug. If this applies to both, then put this in its own heading on the docs at the end of this article.

Reply:
There is a separate community contribution for pulling helm chart from private repository. See PR - https://github.com/dapr/cli/pull/895. It is under review and not a priority for 1.7 it seems. 
Similar to this, the environment variable thing, has implementation only for standalone mode for now and hopefully we will be able to use the same for `dapr init -k` for pulling images from Docker hub/ ghcr.